geomstats/learning/radial_kernel_functions.py

```diff
@@ -0,0 +1,432 @@
+"""Radial kernel functions.
+
+References
+----------
+https://en.wikipedia.org/wiki/Kernel_(statistics)
+https://en.wikipedia.org/wiki/Radial_basis_function
+
+Notes
+-----
+We chose not to apply the normalization coefficients used in some references.
+These normalization coefficients are used in order that the kernel functions
```

Comment:
NIT: "These [/Our??] normalization coefficients were used so that the kernel functions integrate to 1 on the Euclidean space of dimension 1. In contrast, our coefficients...".

-> Which coeffs are you referring to here?

Reply:
The coefficients of the references.
I will change the sentence to make it clearer.